### PR TITLE
Feature/add drop keybind

### PR DESCRIPTION
**Pull request by sarahsturgeon**
## TODO:

 - [x] Test

### CODE REVIEW

File: gamemode/modules/inventory/sv_inventory_interaction.lua [hunk 1]
```diff
@@ -185,3 +195,4 @@ net.Receive( "RVR_Inventory_SetHotbarSelected", function( len, ply )
 
     inv.setSelectedItem( ply, newIndex )
 end )
+
```
Comment (samuelWilliams99) [inline gamemode/modules/inventory/sv_inventory_interaction.lua]: Is this an extra new line?

File: gamemode/modules/inventory/sv_inventory_commands.lua [hunk 2]
```diff
@@ -1,4 +1,4 @@
-local function giveItem( caller, target, item, count )
+function RVR.Commands.giveItem( caller, target, item, count )
     if count < 1 then
```
Comment (samuelWilliams99) [inline gamemode/modules/inventory/sv_inventory_commands.lua]: Theres a point to be made about making these commands global, however they are already global within the Commands module, and this PR is not the place to change this due to the inconsistency it'd cause with other modules.

**Review by samuelWilliams99 — CHANGES_REQUESTED** [review]
Code looks good, just not a fan of the refactor

File: gamemode/modules/inventory/sv_inventory_commands.lua [hunk 3]
```diff
@@ -1,4 +1,4 @@
-local function giveItem( caller, target, item, count )
+function RVR.Commands.giveItem( caller, target, item, count )
```
Comment (samuelWilliams99) [inline gamemode/modules/inventory/sv_inventory_commands.lua]:
```suggestion
local function giveItem( caller, target, item, count )
```

File: gamemode/modules/inventory/sv_inventory_commands.lua [hunk 4]
```diff
@@ -13,8 +13,8 @@ local function giveItem( caller, target, item, count )
     return "Only able to fit " .. amount .. " of " .. count .. " items in inventory."
 end
 
-local function giveSingle( caller, target, item )
-    return giveItem( caller, target, item, 1 )
+function RVR.Commands.giveSingle( caller, target, item )
```
Comment (samuelWilliams99) [inline gamemode/modules/inventory/sv_inventory_commands.lua]:
```suggestion
local function giveSingle( caller, target, item )
```

File: gamemode/modules/inventory/sv_inventory_commands.lua [hunk 5]
```diff
@@ -13,8 +13,8 @@ local function giveItem( caller, target, item, count )
...
+    return RVR.Commands.giveItem( caller, target, item, 1 )
```
Comment (samuelWilliams99) [inline gamemode/modules/inventory/sv_inventory_commands.lua]:
```suggestion
    return giveItem( caller, target, item, 1 )
```

File: gamemode/modules/inventory/sv_inventory_commands.lua [hunk 6]
```diff
@@ -44,7 +44,7 @@ hook.Add( "RVR_ModulesLoaded", "RVR_Inventory_AddCommands", function()
         { "Target", "Item", "Count" },
         { "player", "item", "int" },
         RVR_USER_ADMIN,
-        giveItem,
+        RVR.Commands.giveItem,
```
Comment (samuelWilliams99) [inline gamemode/modules/inventory/sv_inventory_commands.lua]:
```suggestion
        giveItem,
```

File: gamemode/modules/inventory/sv_inventory_commands.lua [hunk 7]
```diff
@@ -53,7 +53,7 @@ hook.Add( "RVR_ModulesLoaded", "RVR_Inventory_AddCommands", function()
         { "Target", "Item" },
         { "player", "item" },
         RVR_USER_ADMIN,
-        giveSingle,
+        RVR.Commands.giveSingle,
```
Comment (samuelWilliams99) [inline gamemode/modules/inventory/sv_inventory_commands.lua]:
```suggestion
        giveSingle,
```

**Review by samuelWilliams99 — COMMENTED** [review]
Suggestion galore

File: gamemode/modules/inventory/sv_inventory_commands.lua (hunk 2 above)
Comment (sarahsturgeon) [reply gamemode/modules/inventory/sv_inventory_commands.lua]: nerd

**Review by samuelWilliams99 — APPROVED** [review]

**Pull request merged**